src/components/Common/SectionTitle.tsx

```diff
@@ -3,7 +3,7 @@ const SectionTitle = ({
   paragraph,
   width = "570px",
   center,
-  mb = "100px",
+  mb = "50px",
```

Comment:
This is a common file. Changes here will affect all pages!

Reply:
I return it the original 100px